apps/codelab/src/app/admin/content/presentation-editor/side-panel/side-panel.component.ts

```diff
@@ -1,46 +1,161 @@
 import {
   Component,
-  EventEmitter,
+  ElementRef,
   HostListener,
   Input,
-  Output
+  OnChanges,
+  OnInit,
+  SimpleChanges
 } from '@angular/core';
 import { Location } from '@angular/common';
-
+import { CdkDragDrop, CdkDragStart, DragRef } from '@angular/cdk/drag-drop';
 import { ActivatedRoute, Router } from '@angular/router';
+
 import { ContentService } from '../services/content.service';
 import { NavigationService } from '../services/navigation.service';
+import { ContentSlide } from '../types';
+
+import { MultiselectModel } from '../../../../multiselect/multiselect-model';
+
+function normalizeSelectionIndexes(indexes: number[]): number[] {
```

Comment:
why is this needed? Add a comment

Reply:
The selection model uses Array.indexOf() (which can return -1, if item not in array) to get selection indexes.
So used this because we are actually do not check that when we select the item it is contained in the SelectionModel.

Probably it would be better to check this case in the model.

Reply:
let's handle it next to Array.indexOf